io.catenax.sealant.sealant_pass/1.0.0/SealantPass.ttl

```diff
@@ -0,0 +1,648 @@
+:documentTitle a bamm:Property;
+    bamm:characteristic :DocumentTitleTrait;
+    bamm:preferredName "document title"@en;
+    bamm:description "The header or title of the document"@en;
+    bamm:exampleValue "Cobald Mine A".
+:documentLocation a bamm:Property;
+    bamm:characteristic bamm-c:Text;
+    bamm:preferredName "document location"@en;
+    bamm:description "Location or link of and to the document. "@en;
+    bamm:exampleValue "ftp://www.company.com/MyCompliance/Compliant02.PDF".
+:sustainabilityReport a bamm:Property;
+    bamm:characteristic :LocateCharacteristic;
+    bamm:preferredName "sustainability report"@en;
+    bamm:description "Link to public website or Document. Reporting in accordance with the new European Sustainability Reporting Standards (ESRS) including the new ESG reporting standards."@en.
+:DocumentTitleTrait a bamm-c:Trait;
+    bamm-c:baseCharacteristic bamm-c:Text;
+    bamm-c:constraint :MaxTitleCharacterConstraint.
+:MaxTitleCharacterConstraint a bamm-c:LengthConstraint;
+    bamm:preferredName "max title character"@en;
+    bamm:description "Maximum title characters set to 60."@en;
+    bamm-c:maxValue "60"^^xsd:nonNegativeInteger;
+    bamm-c:minValue "0"^^xsd:nonNegativeInteger.
+:SpecificsSDSCharacteristic a bamm-c:Set;
+    bamm:dataType :SpecificsSDSEntity;
+    bamm:preferredName "specifics SDS"@en;
+    bamm:description "Connects the SDS version, language, and country. "@en.
+:SpecificsSDSEntity a bamm:Entity;
+    bamm:properties (:documentLanguage :documentVersion :provisioningCountry :documentLocation :otherInformation :manufactureID);
+    bamm:preferredName "specifics SDS"@en;
+    bamm:description "Entity encapsulating the details from the SDS documentation."@en.
+:documentLanguage a bamm:Property;
+    bamm:characteristic bamm-c:Language;
+    bamm:preferredName "document language"@en;
+    bamm:description "Language of the SDS document."@en.
+:documentVersion a bamm:Property;
+    bamm:characteristic :DocumentVersionCharacteristic;
+    bamm:preferredName "document version"@en;
+    bamm:description "Version of the document. "@en.
+:DocumentVersionCharacteristic a bamm-c:Code;
+    bamm:dataType xsd:string;
+    bamm:preferredName "document version"@en;
+    bamm:description "Refers to the version of the SDS document."@en.
+:provisioningCountry a bamm:Property;
+    bamm:characteristic :ProvisioningCountryTrait;
+    bamm:preferredName "provisioning country"@en;
+    bamm:description "The country from where the documentation is issued from. "@en;
+    bamm:exampleValue "GER".
+:ProvisioningCountryCharacteristic a bamm-c:Code;
+    bamm:dataType xsd:string;
+    bamm:preferredName "Provisioning Country"@en;
+    bamm:description "Refers to the country of the SDS provisioner."@en.
+:ProvisioningCountryTrait a bamm-c:Trait;
+    bamm-c:baseCharacteristic :ProvisioningCountryCharacteristic;
+    bamm-c:constraint :ISOCodeConstraint.
+:ISOCodeConstraint a bamm-c:LengthConstraint;
```

Comment:
There was a discussion about having a predefined list of codes in a shared aspect. However, it is not available so might be ok to model it I this way